ajax/libs/videojs-wavesurfer/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "videojs-wavesurfer",
+  "version": "1.3.2",
+  "description": "video.js plugin that adds a navigable waveform for audio and video files.",
+  "filename": "videojs.wavesurfer.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/collab-project/videojs-wavesurfer.git"
+  },
+  "keywords": [
+    "waveform",
+    "audio",
+    "video",
+    "wavesurfer",
+    "videojs",
+    "videojs-plugin",
+    "player"
+  ],
+  "author": "Collab",
+  "license": "MIT",
+  "homepage": "http://videojs.com/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/collab-project/videojs-wavesurfer.git",
+    "fileMap": [
+      {
+        "basePath": "src/js",
```

Comment:
@extend1994 if use `js/*wavesurfer*.+(js|map)` then we create the `js` folder, I don't think this is a good way.
I sanded the PR, which basePath can use `XXX/XXX` as the link --> https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/react-autosuggest/package.json#L38
I think the `basePath` part is fine.